day-2/rabbitmq/application.yaml

```diff
@@ -20,6 +20,10 @@ spec:
       releaseName: rabbitmq
       valueFiles:
         - ./values.yaml
+      service:
+        type: ClusterIP
+        ports:
+          http: 15692
```

Comment:
Helm value changes should be placed in the dedicated file https://github.com/hoprnet/ct-platform/blob/main/day-2-definitions/rabbitmq/values.yaml

Reply:
Why do you need to expose that port?
It should be done if you use the feature of enabling the monitoring

Reply:
I saw that this is the port on which prometheus metrics are exposed